uftrace.c

```diff
@@ -217,13 +218,13 @@ __used static const char uftrace_help[] =
 "\n";
 
 static const char uftrace_shopts[] =
-	"+aA:b:C:d:D:eE:f:F:hH:kK:lL:N:P:r:R:s:S:t:T:U:vVW:Z:";
+	"+aA:b:C:d:D:eE:f:F:hH:kK:lL:N:P:r:R:s:S:t:T:U:vVW:Z";
 
 #define REQ_ARG(name, shopt) { #name, required_argument, 0, shopt }
 #define NO_ARG(name, shopt)  { #name, no_argument, 0, shopt }
 
 static const struct option uftrace_options[] = {
-	REQ_ARG(library-path, 'L'),
+	REQ_ARG(library-path, OPT_library_path),
```

Comment:
I believe not many people directly use `--library-path` option for now, so I would also like to change this option to `--libmcount-path` to be more meaningful. Users might not understand what the library path is.

Reply:
If the purpose of the option is libmcount only, I agree.
I changed it.

Reply:
Do we keep this `--library-path` for backward compatibility?  Then it should be `REQ_ARG(library-path, OPT_libmcount_path)`, isn't it?